tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -130,7 +130,7 @@ install_cached_component() {
 	echo "Downloading tarball from: ${jenkins_build_url}/${component_tarball_name}"
 	curl -fL --progress-bar "${jenkins_build_url}/${component_tarball_name}" -o "${component_tarball_path}" || return cleanup_and_fail
 	curl -fsOL "${jenkins_build_url}/sha256sum-${component_tarball_name}" || return cleanup_and_fail
-	sha256sum -c "sha256sum-${component_tarball_name}" && return cleanup_and_fail
+	sha256sum -c "sha256sum-${component_tarball_name}" || return cleanup_and_fail
```

Comment:
@fidencio we didn't find this error before because it wasn't even reaching to this point?

Reply:
That was my bad, Wainer.

I had this fix as part of a commit that I dropped and I didn't realise that. :-/
When I dropped the commit I was expecting the build to happen from source, what I didn't realise is that I had tat fix squased into the commit that I dropped.